low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -63,14 +64,17 @@ class SSPLBundleGenerate(object):
         self.enclosure_log = f"{SSPL_SB_TMP}/enclosure_logs.zip"
         self.enclosure_wwn = "NA"
 
-    def generate_sspl_support_bundle(self, parser):
-        if os.path.isfile(SSPL_SB_REQUEST):
-            msg = "One SSPL Support Bundle request is ongoing already, skipping current request..!"
+    def generate(self, parser):
+        allow_new_request = validate()
+        if not allow_new_request:
+            msg = "SSPL Support Bundle creation in progress. Existing..."
```

Comment:
seems some typo here @satish-seagate 
needs 'Exiting' or better 'aborting'

take care in any of your future PR

Reply:
Sure Chetan,
Actually, as per ujjwal comments I have made this change. But yes I will take a look into the future patches.
https://github.com/Seagate/cortx-monitor/pull/424#discussion_r653718514